_includes/config/setup-cron_2.3_how-to.md

```diff
@@ -0,0 +1,59 @@
+### Create the Magento crontab
+
+Starting with version 2.2, Magento creates a crontab for you. We add the Magento crontab to any configured crontab for the Magento file system owner. In other words, if you already set up crontabs for other extensions or applications, we add the Magento crontab to it.
+
+The Magento crontab is inside `#~ MAGENTO START` and `#~ MAGENTO END` comments in your crontab.
+
+To create the Magento crontab:
+
+1.	Log in as, or switch to, the [Magento file system owner]({{ page.baseurl }}/install-gde/prereq/file-sys-perms-over.html).
+1.	Change to your Magento installation directory.
+1.	Enter the following command:
+
+    ```bash
+    bin/magento cron:install [--force]
+    ```
+
+Use `--force` to rewrite an existing Magento crontab.
+
+{: .bs-callout-info }
+*	`magento cron:install` does not rewrite an existing crontab inside `#~ MAGENTO START` and `#~ MAGENTO END` comments in your crontab.
+*	`magento cron:install --force` has no effect on any cron jobs outside the Magento comments.
+
+To view the crontab, enter the following command as the Magento file system owner:
+
+```bash
+crontab -l
+```
+
+A sample follows:
+
+```terminal
+#~ MAGENTO START
+* * * * * /usr/bin/php /var/www/html/magento2/bin/magento cron:run | grep -v Ran jobs by schedule >> /var/www/html/magento2/var/log/magento.cron.log
+* * * * * /usr/bin/php /var/www/html/magento2/update/cron.php >> /var/www/html/magento2/var/log/update.cron.log
+* * * * * /usr/bin/php /var/www/html/magento2/bin/magento setup:cron:run >> /var/www/html/magento2/var/log/setup.cron.log
+#~ MAGENTO END
+```
+
+{%
+include note.html
+type='info'
+content='`update/cron.php` does exist if you install Magento using [Composer](https://glossary.magento.com/composer), or [download the archive](https://magento.com/tech-resources/download), in both cases the file exists.
+
+In the case of [Composer](https://glossary.magento.com/composer), the directory `update` and all the files in there are only created when you run `composer create-project`, however if you only run `composer install` you will not see the update directory getting created (if it did not exist before).'
```

Comment:
That's a great tip.
It would really great addition if we have an explanation of how to solve the case when the project already exits but the `update` directory is empty. Since many users might come to check this documentation for the solution where to get the `update/cron.php` in case if it's missing.
Feel free to let me know if you need help with that info.
Thank you

Reply:
@rogyar, I've updated the PR. Could you please review ?

Thank you!